hopsworks-api/src/main/java/io/hops/hopsworks/api/util/VariablesService.java

```diff
@@ -152,5 +152,14 @@ public Response getSecurityQuestions() {
     };
     return Response.ok().entity(questions).build();
   }
+  
+  @GET
+  @Path("isCloud")
+  @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Why is a new endpoint needed? There are similar endpoints there but they shouldn't be anyway. 
I'd suggest we go with using the {id} in the back-end. For the front-end it's ok to use this https://github.com/logicalclocks/hopsworks/pull/519/files#diff-3104bfea9fcc3f42fafed8cc4326a89bR64

Reply:
ok, that should be fine, i was mainly trying not to use the variable name, also didn't want to replicate the logic that checks if the variable value is empty hence it is cloud

Reply:
@tkakantousis {id} is not using the cached values, it is always making a request to the database

Reply:
The expected load from this endpoint is 1 request per week, so NDB can handle it :D
We don't do rate-limiting on the REST API yet, so a DoS attack could equally happen on any other endpoint

Reply:
Ok, will remove it for now